composer.json

```diff
@@ -32,5 +32,5 @@
   },
   "suggest": {},
   "type": "magento2-module",
-  "version": "2.4.100"
+  "version": "2.4.101"
```

Comment:
This gets [automatically bumped by CircleCI](https://github.com/ns8inc/protect-integration-magento/blob/dev/.circleci/config.yml#L30) now, which means...